docs/core/preview3/tools/dotnet-clean.md

```diff
@@ -50,7 +48,6 @@ Defines a configuration under which the build was running.  If omitted, it defau
 
 Defines verbosity to use for the invocation of the `dotnet clean` command. The verbosity levels are standard [MSBuild verbosity levels](https://msdn.microsoft.com/en-us/library/ms164311.aspx). 
```

Comment:
please change the msdn link to https://docs.microsoft.com/visualstudio/msbuild/msbuild-command-line-reference